habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -210,20 +193,29 @@ def initialize_sim(
     backend_cfg.create_renderer = True
     backend_cfg.enable_physics = True
 
-    sensors = {
-        "semantic": {
-            "sensor_type": habitat_sim.SensorType.SEMANTIC,
-            "resolution": [256, 256],
-            "position": [0, 0, 0],
-            "orientation": [0, 0, 0.0],
-        },
-    }
     if debug_viz:
-        sensors["color"] = {
-            "sensor_type": habitat_sim.SensorType.COLOR,
-            "resolution": [256, 256],
-            "position": [0, 0, 0],
-            "orientation": [0, 0, 0.0],
+        sensors = {
+            "semantic": {
+                "sensor_type": habitat_sim.SensorType.SEMANTIC,
+                "resolution": [512, 512],
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0.0],
+            },
+            "color": {
+                "sensor_type": habitat_sim.SensorType.COLOR,
+                "resolution": [512, 512],
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0.0],
+            },
+        }
+    else:
+        sensors = {
```

Comment:
Why create any sensor at all when we aren't in debug mode?

Reply:
The semantic sensor is needed for viewpoints.